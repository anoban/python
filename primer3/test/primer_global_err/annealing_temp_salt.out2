../src/primer3_core: Primer binding does not work with Owczarzy salt correction (PRIMER_SALT_CORRECTIONS=2)
