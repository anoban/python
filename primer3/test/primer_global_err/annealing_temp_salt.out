SEQUENCE_ID=max_annealing_temp
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_TM_FORMULA=1
PRIMER_SALT_CORRECTIONS=2
PRIMER_ANNEALING_TEMP=70.0
PRIMER_ERROR=Primer binding does not work with Owczarzy salt correction (PRIMER_SALT_CORRECTIONS=2)
=
